addons/website_slides/static/src/xml/slide_quiz.xml

```diff
@@ -30,7 +30,7 @@
                         </t>
                         <div class="o_wslides_quiz_answer_info list-group-item list-group-item-info d-none">
                             <i class="fa fa-info-circle"/>
-                            <span class="o_wslides_quiz_answer_comment"/>
+                            <span class="o_wslides_quiz_answer_comment ms-1"/>
```

Comment:
Hello @pyduf , I was just wondering while we are at it, do you also have this weird padding delta with other answers? 

In the bottom question, I've added a ms-2 instead.

![image](https://github.com/odoo/odoo/assets/71645720/ca3f94cd-e29a-483c-a619-d05fda07f6f7)

There is also a delta when you leave the fullscreen. (lesson_content_quiz_question template?) Maybe we could add ms-1 there? Here, in the top question I've added ms-1

![image](https://github.com/odoo/odoo/assets/71645720/112895b2-434c-4684-9006-d673f7a84063)

What do you think? Cheers!

Reply:
Hello @Odoonan, 

Thanks for the review and the suggestion. As explained in the commit, it doesn’t solve the root cause which seems to be an optimization that remove all spaces between tags since the version 17 (html/css has not changed but the look well). This only happens in the full screen, that’s why I added “ms-1” and not “ms-2” to have the same look as in the “not full screen”. But I hesitated to use “ms-2” because it feels better aligned as you said so I have modified the code so that in full screen I use “ms-2” and in “not full screen” “ms-1” to have the same look in both as well but with a better alignment. Other spacing in the quiz seems right.

Note that the runbot is red but only because of unrelated known errors. I will rebase it later to get it green.

Reply:
Hello @pyduf , thanks for the explanations :) 
I'm greening then. 
Cheers!